source/common/http/conn_manager_impl.h

```diff
@@ -388,26 +388,67 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     NullRouteConfigUpdateRequester() = default;
   };
 
+  /**
+   * Callbacks invoked by the FilterManager to pass filter data/events back to the caller.
+   */
   class FilterManagerCallbacks {
   public:
     virtual ~FilterManagerCallbacks() = default;
 
-    // The resulting data will be provided through the following callbacks.
+    /**
+     * Called with the fully encoded response headers.
+     * @param response_headers the encoded headers.
+     * @param end_stream whether this is a header only response.
+     */
     virtual void encodeHeaders(ResponseHeaderMap& response_headers, bool end_stream) PURE;
+
+    /**
+     * Called with the fully encoded 100 Continue response headers.
```

Comment:
Fully encoded here isn't totally clear IMO
Called when YY has been encoded by all filter chains?  Called when it's ready to encode to the codec?